src/java.base/share/classes/java/lang/classfile/ClassTransform.java

```diff
@@ -156,11 +160,22 @@ static ClassTransform transformingFields(FieldTransform xform) {
         return new TransformImpl.ClassFieldTransform(xform, f -> true);
     }
 
+    /**
+     * @implSpec
```

Comment:
Should these overridden method have `@inheritDoc` ? If not, their javadoc won't show much, I believe.

Reply:
(To be precise - the javadoc says "Description copied from interface: ClassFileTransform") - just make sure that's what you want.

Reply:
The javadoc shows what is expected, so I'm not sure `@inheritDoc` is still required.